### DISCUSSION TITLE
SDK v2

### DISCUSSION THREAD

**Discussion by dregade** [AxisCommunications/discussions#156] [ACAP] [answered]
Hi everyone,

Can someone please tell me where I can download sdk v2 ?
I need to compile my app  for mips and some cameras with firmware lower than 9.70 (8.x and 6.50.x).

Thanks

**Comment by mattias-kindborg-at-work** [comment, marked as answer]
Hi @dregade. ACAP version 2 is unfortunately no longer supported.

**Comment by FerranLisa** [comment]
HI Mattias,
Does that mean that using ACAP with old Axis cameras with MIPS is not possible?
I have a customer asking me to develop an app for a network of Q1615-MKii cameras.
The app is pretty simple, sohuld be easy to implement in MIPS, but I need some SDK.
Please revert,
Ferran

**Comment by onie5** [comment]
Hola  Ferran,

I still have the old ACAP2 SDK package.

**Reply by FerranLisa** [reply]
Hi Onie5
Could you please share it with me?
Best regards,
Ferran

**Reply by sruizvargasp3** [reply]
También tengo el bin del SDK2
Y también una imagen que AXIS proporcionaba, para hacer una maquina virtual con virtualbox. Esa maquina virtual ya tenía todo instalado, creo que también el compilador para mips.
Dime si todavía necesitas
